megapowerani.com.template.json

```diff
@@ -0,0 +1,70 @@
+{
+    "providerId": "megapowerani.com",
+    "providerName": "Freshsales Test",
```

Comment:
`Test` in Provider name seems to be confusing. 
If this is a test template I would rather see it in `serviceId` / `serviceDescription`.
Note that most of DNS providers would display this name in their UI.

Reply:
@pawel-kow Apologies, I will perform the changes where I mention "Test" in the serviceID/serviceDescription. I am performing this test so that the team who will work on the production application can create an new production template later. I am only preparing this for a demo so that this gets accepted for production.